package.json

```diff
@@ -49,6 +49,7 @@
     "react-scripts": "5.0.1",
     "react-select": "^5.7.3",
     "react-select-event": "^5.5.1",
+    "react-switch": "^7.0.0",
```

Comment:
This is not actually the switch component documented at https://mui.com/material-ui/react-switch/ that I thought we were using. This is a completely different package documented at https://www.npmjs.com/package/react-switch#usage which has about the same API.

This one might be a little better, so maybe we should keep it?